caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConnection.java

```diff
@@ -94,13 +94,18 @@ private static int getNumberFromSysPropertyOrEnvVariable(final String environmen
 
     @Override
     public AuditChannel createChannel() throws IOException {
+        if (!isIndexTemplateCreated || isForceIndexTemplateUpdate) {
```

Comment:
If CAF_AUDIT_FORCE_INDEX_TEMPLATE_UPDATE is true is the intention that every call to createChannel will cause the template to be created?